src/components/molecules/ScheduleNG/ScheduleNG.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from "react";
+import classNames from "classnames";
+
+import { PersonalizedVenueEvent } from "types/venues";
+
+import { eventEndTime, eventStartTime } from "utils/event";
+import { formatDateRelativeToNow, formatTimeLocalised } from "utils/time";
+
+import { Loading } from "components/molecules/Loading";
+
+import "./ScheduleNG.scss";
+
+export interface ScheduleNGProps {
+  daysEvents: PersonalizedVenueEvent[];
+  isLoading: boolean;
+}
+
+export const ScheduleNG: React.FC<ScheduleNGProps> = ({
+  daysEvents,
+  isLoading,
+}) => {
+  const hasEvents = daysEvents.length > 0;
+
+  const containerClasses = classNames("ScheduleNG");
+
+  const rowsWithTheEvents = useMemo(() => {
+    const formatDateOptions = { formatToday: () => "" };
+
+    return daysEvents.map((event) => (
+      <div key={event.id} className="ScheduleNG__item">
+        <div className="ScheduleNG__item--info">
+          <span className="ScheduleNG__item--date">
+            {formatDateRelativeToNow(eventStartTime(event), formatDateOptions)}
+          </span>
+
+          <span className="ScheduleNG__item--time">
+            {formatTimeLocalised(eventStartTime(event))}
+          </span>
+
+          <span className="ScheduleNG__item--date">
+            {formatDateRelativeToNow(eventEndTime(event), formatDateOptions)}
+          </span>
+
+          <span className="ScheduleNG__item--time">
+            {formatTimeLocalised(eventEndTime(event))}
+          </span>
+        </div>
+        <div className="ScheduleNG__item--name">{event.name}</div>
+      </div>
```

Comment:
can we create a separate component for the event line which other engineers will use for their tasks?

Reply:
created `ScheduleItemNG` as a separate component and fixed in 03694e555e05deb1888ddfa97d6c16717799922b